### PR TITLE
Setup CI

### PR DESCRIPTION
**Pull request by sgfn**

### CODE REVIEW

**Review by mickel8 — COMMENTED** [review]
Awesome work!

Could we just check whether it's possible to reset cache?

File: broadcaster/lib/broadcaster_web/controllers/media_controller.ex [hunk 1]
```diff
@@ -53,7 +53,7 @@ defmodule BroadcasterWeb.MediaController do
       :ok = PeerConnection.add_ice_candidate(pid, candidate)
       resp(conn, 204, "")
     else
-      {:error, _} -> resp(conn, 400, "Bad request")
+      _other -> resp(conn, 400, "Bad request")
```
[the remark this answers is not in the document]
Comment (sgfn) [reply broadcaster/lib/broadcaster_web/controllers/media_controller.ex:56]: Some functions returned `{:error, reason}`, others simply `:error`, so credo complained

File: .github/workflows/build-check-app.yml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      if: ${{ inputs.with-ffmpeg }}
+      working-directory: .
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y wget tar
+        wget ${{ env.FFMPEG_URL }} -O - | tar xJ
+        sudo mv ffmpeg-* /opt/ffmpeg
+        echo "C_INCLUDE_PATH=/opt/ffmpeg/include" >> $GITHUB_ENV
+        echo "LIBRARY_PATH=/opt/ffmpeg/lib" >> $GITHUB_ENV
+        echo "LD_LIBRARY_PATH=/opt/ffmpeg/lib" >> $GITHUB_ENV
```
[the remark this answers is not in the document]
Comment (sgfn) [reply .github/workflows/build-check-app.yml:47]: Oh the hypocrisy

```
==> xav
Xav requires FFmpeg development packages to be installed on your system. Make sure paths to ffmpeg header and library files are in C_INCLUDE_PATH, LIBRARY_PATH and LD_LIBRARY_PATH. For more information refer to Xav README under: https://github.com/mickel8/xav#installation
```

[the remark this answers is not in the document]
Comment (sgfn) [reply .github/workflows/build-check-app.yml:47]: Just a joke
However, I think I've seen some errors when compiling Xav on the CI runner that disappeared after setting `LD_LIBRARY_PATH`. Unfortunately I deleted the logs

**Review by LVala — APPROVED** [review]

File: .github/workflows/build-check-app.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mickel8) [reply .github/workflows/build-check-app.yml:47]: Yeah this log in Xav was added before our research :D 

Okay, let's merge this as it is. I am going to add support for older FFmpegs in Xav so we can remove this in the future anyway

**Pull request merged**